### ISSUE TITLE
Avoid stdout/stderr implicit writing

### ISSUE THREAD

**Issue by alberto743** [GEOUNED-org/GEOUNED#143]
GEOUNED should not dump anything to stdout/stderr, unless the loading script requires this by a configurable parameters.
This will allow employing GEOUNED as a library for further processing.
For instance, the status bar provided by `tqdm` may be activated only by a specific input parameter.

**Comment by alberto743** [comment]
Yes, an argument to the `start()` method would be fine for me, with default set to `False`.

**Comment by alexvalentine94** [comment]
For now add the argument to start()
